src/octoprint/plugins/softwareupdate/__init__.py

```diff
@@ -324,8 +325,8 @@ def on_settings_save(self, data):
 
 		if "octoprint_type" in data:
 			octoprint_type = data["octoprint_type"]
-
-			if octoprint_type == "github_release":
+			self._logger.debug("octoprint data - {}".format(data))
```

Comment:
This log line is
- not descriptive enough
- unnecessary most of the time?

Reply:
this was added for debugging, I will remove it